common/src/main/java/io/netty/util/internal/PlatformDependent0.java

```diff
@@ -845,14 +847,38 @@ private static int javaVersion0() {
         if (isAndroid0()) {
             majorVersion = 6;
         } else {
-            majorVersion = majorVersionFromJavaSpecificationVersion();
+            majorVersion = majorVersionFromSystemClass();
         }
 
         logger.debug("Java version: {}", majorVersion);
 
         return majorVersion;
     }
 
+    // https://docs.oracle.com/javase/specs/jvms/se7/html/jvms-4.html
```

Comment:
what does this comment mean?  It doesn't seem to have any relation to the parsing of the version.